.github/config.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Heavily inspired by https://github.com/electron/electron/blob/master/.github/config.yml
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
```

Comment:
You may also want remove the part about the steps to reproduce here and just mention that they should follow the issue template (which includes the steps to reproduce).

Reply:
I think it would be good to keep because it's less likely to get ignored as it could be if it's in a link.